CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ Semantic versioning in our case means:
 - WPS531: Forbids testing conditions to just return booleans when it is possible to simply return the condition itself
 - Forbids to use unsafe infinite loops
 - Forbids to use raw strings `r''` when not necessary
+- Add parameter to `given_function_called` so that it checks if a function has been called by its name regardless of the modules or classes it is in
```

Comment:
Btw, we only put user-facing changes here. Som you can delete this line in the future PR.

Reply:
ah, gotcha